src/main/java/de/mediathekview/mlib/compression/CompressionManager.java

```diff
@@ -41,7 +40,7 @@ public OutputStream compress(final CompressionType aCompressionType,
       final OutputStream aOutputStream) throws IOException {
     switch (aCompressionType) {
       case XZ:
-        return new XZOutputStream(aOutputStream, new LZMA2Options(LZMA2Options.PRESET_MAX));
+        return new XZCompressorOutputStream(aOutputStream, 9);
       case GZIP:
         return new GZIPOutputStream(aOutputStream) {
           {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the contents of this initializer to a standard constructor or to field initializers. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1171)

Reply:
1st) There is no Enum inside the Apache Framework to get an readable noun for "strongest Compression". So the 9 is good enough here. 2nd) it is a update to the dependencies not a general refactoring of the code so no refactoring where made. Put a new Request for Refactoring.